src/stylesheets/core/_variables.scss

```diff
@@ -774,6 +774,7 @@ $focus-outline:                 units(0.5) solid $color-focus !default;
 $focus-spacing:                 0 !default;
 $nav-width:                     951px !default;
 $sidenav-current-border-width:  0.4rem !default; // must be in rem for math
+$search-min-width:              27em !default; // minimum width of 27 letters for comfortable search bar
```

Comment:
I've already approved (and I like the way it looks right now) but perhaps `27ch` better reflects the 27-character guidance?